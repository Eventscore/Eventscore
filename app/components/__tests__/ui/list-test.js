/**
 * Test to check if the component renders correctly
 */
/* global it expect */
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import { List, ListItem } from '@ui';

it('List renders correctly', () => {
  const tree = renderer.create(
    <List>
      <ListItem title={'Hello'} />
      <ListItem title={'Second'} />
    </List>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
