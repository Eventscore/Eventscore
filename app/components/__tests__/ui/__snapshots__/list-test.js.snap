// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "borderBottomWidth": 1,
        "borderColor": "#cbd2d9",
        "borderTopWidth": 1,
        "marginTop": 20,
      },
      Array [
        Object {
          "backgroundColor": "#E9EBEE",
          "borderBottomWidth": 0,
          "borderTopColor": "#D0D1D5",
          "margin": 0,
        },
        Array [],
      ],
    ]
  }
>
  <View
    onLongPress={undefined}
    onPress={undefined}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderBottomColor": "#ededed",
          "borderBottomWidth": 1,
          "paddingBottom": 10,
          "paddingRight": 10,
          "paddingTop": 10,
        },
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "borderBottomColor": "#D0D1D5",
            "borderTopColor": "#D0D1D5",
          },
          Array [],
        ],
      ]
    }
    underlayColor="#D0D1D5"
  >
    <View
      style={
        Array [
          Object {
            "flexDirection": "row",
            "marginLeft": 10,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <View
          style={undefined}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "fontFamily": undefined,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "#43484d",
                    "fontSize": 17.5,
                  },
                  Object {
                    "marginLeft": 10,
                  },
                  Array [
                    Object {
                      "color": "#222222",
                      "fontFamily": "HelveticaNeue",
                      "fontSize": 14,
                      "fontWeight": "500",
                      "lineHeight": 18,
                    },
                    Array [],
                  ],
                  undefined,
                ],
              ]
            }
          >
            Hello
          </Text>
        </View>
        <View
          style={undefined}
        >
          <View />
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flex": 0.15,
            "justifyContent": "center",
          }
        }
      >
        <View
          onLongPress={undefined}
          onPress={undefined}
          style={
            Array [
              false,
              false,
              false,
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
          underlayColor="white"
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#777777",
                  "fontSize": 28,
                },
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Array [
                    Object {},
                    undefined,
                  ],
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    onLongPress={undefined}
    onPress={undefined}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderBottomColor": "#ededed",
          "borderBottomWidth": 1,
          "paddingBottom": 10,
          "paddingRight": 10,
          "paddingTop": 10,
        },
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "borderBottomColor": "#D0D1D5",
            "borderTopColor": "#D0D1D5",
          },
          Array [],
        ],
      ]
    }
    underlayColor="#D0D1D5"
  >
    <View
      style={
        Array [
          Object {
            "flexDirection": "row",
            "marginLeft": 10,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <View
          style={undefined}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "fontFamily": undefined,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "#43484d",
                    "fontSize": 17.5,
                  },
                  Object {
                    "marginLeft": 10,
                  },
                  Array [
                    Object {
                      "color": "#222222",
                      "fontFamily": "HelveticaNeue",
                      "fontSize": 14,
                      "fontWeight": "500",
                      "lineHeight": 18,
                    },
                    Array [],
                  ],
                  undefined,
                ],
              ]
            }
          >
            Second
          </Text>
        </View>
        <View
          style={undefined}
        >
          <View />
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flex": 0.15,
            "justifyContent": "center",
          }
        }
      >
        <View
          onLongPress={undefined}
          onPress={undefined}
          style={
            Array [
              false,
              false,
              false,
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
          underlayColor="white"
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#777777",
                  "fontSize": 28,
                },
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Array [
                    Object {},
                    undefined,
                  ],
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
